app/src/androidTest/java/com/ui/espresso/screens/TypeText.kt

```diff
@@ -0,0 +1,49 @@
+package com.ui.espresso.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import com.ui.espresso.R
+import com.ui.espresso.constants.Strings
+import org.hamcrest.CoreMatchers.not
+
+object TypeText {
+
+    // region | Constants --------------------------------------------------------------------------
+    private const val NAME_EDITTEXT: Int = R.id.name_edittext
+    private const val ENTER_NAME: Int = R.string.enter_name
+    private const val ERROR_TEXT: Int = R.id.error_text
+    private const val NEXT_BUTTON: Int = R.id.next_button
+    private const val GREETING_MESSAGE: Int = R.id.greeting_message
```

Comment:
Remove the `Int` type requirement on the variables.

Reply:
Done